sep490-enterprise/src/main/resources/application.yml

```
@@ -0,0 +1,40 @@
+logging:
+  level:
+    root: INFO
+    org:
+      springframework:
+        security: ${SPRING_SECURITY_LOG_LEVEL:INFO}
+        web: ${SPRING_WEB_LOG_LEVEL:INFO}
+      hibernate:
+        SQL: ${HIBERNATE_SQL_LOG_LEVEL:TRACE}
+        type:
+          descriptor:
+            sql: ${HIBERNATE_SQL_PARAM_LOG_LEVEL:TRACE}
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)